src/index.html

```diff
@@ -11,12 +11,103 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="main-header">
+      <a
+        href="#logo"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo company logo, showing the company name in blue and white"
+        />
+      </a>
+
+      <nav>
+        <ul class="nav-menu">
+          <li>
+            <a
+              href="#apple"
+              class="nav-link is-active"
+            >
+              apple
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class `nav-link` is good, but ensure that all class names follow this pattern.